guides/release/models/finding-records.md

```diff
@@ -2,34 +2,36 @@ The EmberData store provides an interface for retrieving records of a single typ
 
 ### Retrieving a Single Record
 
-Use [`store.findRecord()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/findRecord?anchor=findRecord) to retrieve a record by its type and ID.
-This will return a promise that fulfills with the requested record:
+Use [`findRecord()`](https://api.emberjs.com/ember-data/5.3/functions/@ember-data%2Fjson-api%2Frequest/findRecord) to retrieve a record by its type and ID.
```

Comment:
I think this should likely still be `release` vs `5.3` here

Reply:
Yes. Changed it.